features/exploredash/src/main/res/drawable/ic_compass.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="20dp"
+    android:viewportHeight="20" android:viewportWidth="20"
+    android:width="20dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#008DE4" android:pathData="M10.0007,6.6666C8.159,6.6666 6.6673,8.1582 6.6673,9.9999C6.6673,11.8416 8.159,13.3333 10.0007,13.3333C11.8423,13.3333 13.334,11.8416 13.334,9.9999C13.334,8.1582 11.8423,6.6666 10.0007,6.6666ZM17.4506,9.1666C17.0673,5.6916 14.309,2.9332 10.834,2.5499V0.8333H9.1673V2.5499C5.6923,2.9332 2.934,5.6916 2.5506,9.1666H0.834V10.8333H2.5506C2.934,14.3083 5.6923,17.0666 9.1673,17.4499V19.1666H10.834V17.4499C14.309,17.0666 17.0673,14.3083 17.4506,10.8333H19.1673V9.1666H17.4506ZM10.0007,15.8333C6.7757,15.8333 4.1673,13.2249 4.1673,9.9999C4.1673,6.7749 6.7757,4.1666 10.0007,4.1666C13.2256,4.1666 15.834,6.7749 15.834,9.9999C15.834,13.2249 13.2256,15.8333 10.0007,15.8333Z"/>
+</vector>
```

Comment:
This icon is already added as `ic_current_location`, you can reuse that instead by applying a blue tint to it.